UPGRADE-4.0.md

```diff
@@ -452,6 +452,32 @@ HttpFoundation
 HttpKernel
 ----------
 
+ * Relying on convention-based commands discovery is not supported anymore.
+   Use PSR-4 based Service Discovery instead.
+
+   Before:
+
+   ```yml
+   # app/config/services.yml
+   services:
+       # ...
+
+       # implicit commands registration
```

Comment:
maybe add "implicit registration of all commands in the `Command` folder" for clarity